index.html

```diff
@@ -1004,16 +1004,21 @@ <h2>Basic Concepts</h2>
         <h3>Contexts</h3>
 
         <p>
-When two software systems need to exchange data, they must use terminology and
-a protocol that both systems understand. As an analogy, consider how two
-people communicate. Both people must use the same language and the words they
-use must mean the same thing to each other. This is often referred to as
-"the context in which a conversation happens". This specification defines the
-use of the <code>@context</code> <a>property</a> when communicating information
-related to specific <a>verifiable credentials</a> or
-<a>verifiable presentations</a>.
-        </p>
-
+When two software systems need to exchange data, they need to use terminology that both
+systems understand. (They also need to use a shared protocol; one or more protocols for
```

Comment:
recommend removing the parenthetical comment.

Reply:
Agreed, it'll ruin the surprise. :P 

It also adds more than necessary to get the point across.

Reply:
Happy to do that. The only reason that the protocol part is there is because protocol was there in the original text. But it is not needed in the revised text in my opinion.